plugins/Denizen/scripts/climbing_chalk.dsc

```diff
@@ -0,0 +1,79 @@
+climbing_chalk:
+    type: item
+    debug: false
+    material: sugar
+    allow in material recipes: false
+    display name: <dark_aqua>Climbing Chalk
+    lore:
+    - <&7>Strengthens your grip.
```

Comment:
Make this a different color, as it's sort of like the item "motto" instead of the actual description